manim/logger.py

```diff
@@ -30,9 +30,14 @@ def parse_theme(fp):
     theme["log.height"] = (
         None if theme["log.height"] == "-1" else int(theme["log.height"])
     )
+    theme["log.timestamps"] = False if theme["log.timestamps"] == "False" else True
```

Comment:
Suggestion: prefer `theme['log.timestamps'] = theme.getboolean('log_timestamps')`.

Reply:
`theme` is actually a dictionary. I've done `theme['log.timestamps'] = config_parser["logger"].getboolean('log.timestamps')` instead in the latest commit.